### PR TITLE
VTCCA-4709 added aria lable to the link

### PR DESCRIPTION
**Pull request by zahiriqbal**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4709

### CODE REVIEW

File: app/views/propertyrepresentation/manage/myAgents.scala.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h1 class="govuk-heading-l">@messages("propertyRepresentation.showAgents.title")
                     content = Text(messages("propertyRepresentation.showAgents.properties", agent.propertyCount, propertyLinkingCount))
                 ),
                 TableRow(
-                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}")))
+                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}", s"View agent ${agent.name} (${agent.representativeCode})")))
```
Comment (darrell-plant) [inline app/views/propertyrepresentation/manage/myAgents.scala.html:45]: either move `View agent` text into the messages file pls (text should be in messages file)
so
```
s"${agent.name} (${agent.representativeCode})
```

File: conf/messages [hunk 2]
```diff
@@ -560,7 +560,7 @@ propertyRepresentation.agentProperties.assigned.title=Your agent
 propertyRepresentation.agentProperties.assigned=Assigned properties
 propertyRepresentation.manage.agent=Manage this agent
 propertyRepresentation.showAgents.properties={0} of {1} properties
-propertyRepresentation.showAgents.viewAgentDProperties=<a class="govuk-link" href="{0}">View</a>
+propertyRepresentation.showAgents.viewAgentDProperties=<a class="govuk-link" href="{0}" aria-label="{1}">View</a>
```
Comment (darrell-plant) [inline conf/messages:563]: how about fixing this typo too

```
viewAgentDProperties
```
should be
```
viewAgentProperties
```

Comment (darrell-plant) [reply conf/messages:563]: make sure it's not used anywhere else

**Comment by darrell-plant** [comment]
LGTDz 👍

File: app/views/propertyrepresentation/manage/myAgents.scala.html [hunk 3]
```diff
@@ -42,7 +42,7 @@ <h1 class="govuk-heading-l">@messages("propertyRepresentation.showAgents.title")
                     content = Text(messages("propertyRepresentation.showAgents.properties", agent.propertyCount, propertyLinkingCount))
                 ),
                 TableRow(
-                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentDProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}")))
+                    content = HtmlContent(Html(messages("propertyRepresentation.showAgents.viewAgentProperties", s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}", s"${agent.name} (${agent.representativeCode})")))
```
Comment (PeterPerhac) [inline app/views/propertyrepresentation/manage/myAgents.scala.html:45]: unnecessary string interpolation here
`s"${controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url}"`
is same as
`controllers.agent.routes.ManageAgentController.manageAgentProperties(agent.representativeCode, GetPropertyLinksParameters(), PaginationParameters(1, 15)).url`

**Comment by PeterPerhac** [comment]
👍  small nitpick. would be nice to improve but not a show stopper

**Pull request merged**